consensus/istanbul/backend/engine.go

```diff
@@ -289,8 +289,8 @@ func (sb *backend) verifyCommittedSeals(chain consensus.ChainReader, header *typ
 		}
 	}
 
-	// The length of validSeal should be larger than number of faulty node + 1
```

Comment:
In the master the condition is 
`if validSeal < 2*snap.ValSet.F()`
shouldn't it be this instead? 
`if validSeal <= snap.ValSet.F()`